src/io/flutter/inspector/DiagnosticsTreeStyle.java

```diff
@@ -72,13 +72,15 @@
   singleLine,
 
   /**
-   * Render the tree on a single line with the name and value on separate
-   * lines.
+   * Render the tree on a single line with the name and value on separate lines.
```

Comment:
this name is obsolete. as part of the code review of the matching Flutter code it changed to errrorProperty.
https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/foundation/diagnostics.dart#L163

Reply:
gotcha, I’ll update this file